utils/update_build_version.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Shaderc Authors. All rights reserved.
```

Comment:
This line causes the copyright check to fail, and so my default build fails.
You already filed https://github.com/google/shaderc/issues/139 for that issue.